src/sentry/analytics/event_manager.py

```diff
@@ -1,20 +1,22 @@
 __all__ = ("default_manager", "EventManager")
 
+from typing import Any, MutableMapping, Type
+
 from sentry.analytics.event import Event
 
 
 class EventManager:
-    def __init__(self):
-        self._event_types = {}
+    def __init__(self) -> None:
+        self._event_types: MutableMapping[Any, Type[Event]] = {}
 
-    def register(self, event_cls: Event) -> None:
+    def register(self, event_cls: Type[Event]) -> None:
```

Comment:
Another question: How does `: Type[Event]` vary from `: Event` and should we be using `Type[Event]` from now on?

Reply:
```py
class Event:
  pass

event = Event()
type(event) == Event  # True
type(type(event)) == type(Event)  # True
type(Event)  # <class 'type'>
```
In this example, `event` is an instance and `Event` is a class. Most of our functions deal with instances because they actually contain data. The `register()` function is actually trying to use the _classes_ as a parameter so that it can register them into the `_event_types` table.

Reply:
I see, so this syntax is only used in this case because `event_cls` isn't an instance of an event, but the actual class instead. Thanks!